lib/msf/util/document_generator/pull_request_finder.rb

```diff
@@ -81,6 +81,8 @@ def get_commits_from_file(path)
             commits = git_client.commits(repository, branch, path: path)
           rescue Faraday::ConnectionFailed
             raise PullRequestFinder::Exception, 'No network connection to Github.'
+          rescue Octokit::Unauthorized
+            raise PullRequestFinder::Exception, 'Authentication Failed #{e}.'
```

Comment:
Looks like this needs a tweak 👀 

<img width="918" alt="image" src="https://user-images.githubusercontent.com/60357436/156463146-1fa95526-1f25-40e2-a36e-b2e39cdd1561.png">

Reply:
Woooops.  I was printing the exception to find out what it was and I forgot to remove it once I figured it out....